envs/openwrt/shell.nix

```diff
@@ -21,6 +21,8 @@ let
       unzip
       utillinux
       python2
+      python3.6
```

Comment:
Interesting that it has to be exactly 3.6.

Reply:
It asked me for Python 3.6, so I added exactly that version.

Reply:
At least in their official guide they are happy to use latest python in archlinux (which should be 3.9?): https://openwrt.org/docs/guide-developer/build-system/install-buildsystem
The only issue I see is that python3.6 will go end-of-live an than our `shell.nix` needs an update.